dev-tools/reconfigurator-cli/tests/input/cmds-target-release.txt

```diff
@@ -31,46 +31,78 @@ sled-list
 blueprint-list
 inventory-list
 
-# First step: upgrade one SP.
+# First step: upgrade one RoT.
 blueprint-plan dbcbd3d6-41ff-48ae-ac0b-1becc9b2fd21 f45ba181-4b56-42cc-a762-874d90184a43
 blueprint-diff dbcbd3d6-41ff-48ae-ac0b-1becc9b2fd21 8da82a8e-bf97-4fbd-8ddd-9f6462732cf1
```

Comment:
Similarly, all of these can be just

```suggestion
blueprint-diff latest
```

for "diff the latest blueprint against its parent".